.browserslistrc

```diff
@@ -1,8 +1,17 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
+# For the full list of supported browsers by the Angular framework, please see:
+# https://angular.io/guide/browser-support
+
 # You can see what browsers were selected by your queries by running:
 #   npx browserslist
 
-> 0.5%
-last 2 versions
+last 1 Chrome version
+last 1 Firefox version
+last 2 Edge major versions
+last 2 Safari major versions
+last 2 iOS major versions
```

Comment:
Why was this file changed? Is it part of `ng update`?

Reply:
I updated this file after the upgrade to ng 11 as it was issuing warnings relating to IE 9/10 no longer being supported. I just copied this file from a new ng 11 project. happy to roll this one back

Reply:
I think it should be reverted since the module does still work down to IE9. I assume that Angular removed it, because you'd need extra polyfills to support IE.

Reply:
I have reverted this file